frontend/src/api/auth.ts

```diff
@@ -0,0 +1,7 @@
+import root from "./root";
+
+export default class AuthApi {
+  public static login() {
+    return root.get("/logs/");
```

Comment:
Hmm... assuming this is a placeholder? If so, let's comment to that effect in the file so people aren't guessing about what "logs" this is showing.

Reply:
Yes, it's a placeholder. I will make that more clear.